docs/concepts/workloads/pods/init-containers.md

```diff
@@ -129,6 +129,30 @@ spec:
     command: ['sh', '-c', 'echo The app is running! && sleep 3600']
 ```
 
+There is a slight change of syntax in Kubernetes version 1.6.
+We move declaration of the init containers to `spec`:
```

Comment:
In Kubernetes 1.6, init-containers were made a field in the API. The beta annotation is still respected but will be deprecated in future releases.